.github/workflows/merge-release-tag.yml

```diff
@@ -0,0 +1,54 @@
+name: Merge Release Tag
+
+on:
+  push:
+    tags:
+      - 'mongosh@[0-9]+.[0-9]+.[0-9]+'
```

Comment:
This assumes we don't do prereleases - is this a safe assumption to make?

Reply:
I'll make it a bit more permissive just in case